src/compiler/program.ts

```diff
@@ -396,7 +396,7 @@ export function createGetSourceFile(
         }
         catch (e) {
             if (onError) {
-                onError(e.message);
+                onError(e instanceof Error ? e.message : `${e}`);

```

Comment:
two nits that you are free to ignore, since I'm talking about edge cases here:
1. `instanceof` is unreliable in the JS ecosystem when we try to check things that we don't control. It's possible to have an instance of an `Error` from a different realm. Duck-typing might be more reliable when it comes to checking if something is an Error (or Error-like)
2. interpolating `e` into a template literal **might** throw if `typeof e === 'symbol'`

Reply:
Yeah, these are valid; I mainly did this just to see if I found any bugs with the option reenabled (probably one!), but I agree these aren't ideal. We do have some existing instanceof checks like these. These are largely best-effort, though, so I'm not majorly concerned, but it is valid.